Sources/ProjectDescription/Dependencies/Dependencies.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+public struct Dependencies: Codable, Equatable {
```

Comment:
This is a user-facing model and therefore should be documented. Otherwise, developers won't get documentation when editing the manifest.